packages/DataGrid/src/DataGrid.js

```diff
@@ -289,16 +290,6 @@ const DataGrid = React.forwardRef((props, ref) => {
     [refScrollGrid]
   );
 
-  const handleRefCell = React.useCallback(({ columnIndex, rowIndex }) => {
```

Comment:
`handleRefCell` was used to have access to the cell imperative API, but was really heavy, I think there can be a better alternative to change just one cell or a number of cell without having to store the ref or each cell. Maybe recoil can work better for this specific case.

Again this is a feature no currently use by any DataGrid, so is fine to remove it and thinking about again.